org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -448,8 +450,8 @@ class Translator {
             .then(function (body) {
                 return JSON.parse(body);
             })
-            .catch((err) => {
-                console.log("Request failed to: " + options.method + " - " + options.url);
+            .catch((err) => {                
+                this.ConsoleLogger.error("Request failed to: ${options.method} - ${options.url}"); 
```

Comment:
Templates work wtih the back-tick i believe.. I dont think this would work -- surprised tests didnt catch this -- can you see if there are tests that invoke this path in teh unit tests today? If not can you help add one or file an issue to create a test and then add it?